src/App.js

```diff
@@ -1,17 +1,118 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {createStore} from 'redux';
 
-var HelloWorld = React.createClass({
-  render: function() {
-    return (
-      <div>
-        <h1>Hello World!</h1>
-      </div>
-    )
-  }
+
+
+const defaultState = {list: []}
```

Comment:
you should have separated this file out into multiple other files. It'll start to get hard tracking and reading your code once your project gets bigger.